packages/extension-chakra-storefront/src/setup-app.tsx

```diff
@@ -49,7 +51,7 @@ class ChakraStorefront extends ApplicationExtension<Config> {
         return applyHOCs(App, requiredHOCs)
     }
 
-    getRoutes(): Promise<RouteProps[]> {
+    async getRoutes(): Promise<RouteProps[]> {
```

Comment:
I don't think we need to make this async.. .and thus we wouldn't need a definition for the `getComponentMap` method, nor import the pages.

Reply:
good point! I'l leave it for now so we can test that everything is working with an async `getRoutes` but will revert before merging